composer.json

```diff
@@ -18,6 +18,7 @@
         "drupal/devel": "~1.2",
         "drupal/drupal-extension": "~4.0",
         "drush/drush": "~9.0@stable",
+        "egulias/email-validator": "^1.2.1",
```

Comment:
I have concerns about this.

First, if adding this line is fixing a bigger issue with user forms, then it's out of the scope of this and should be on its own sub ticket and this ticket should depend on it.

Reply:
Isnt this for the lowest dependency thing? @drupol @imanoleguskiza

Reply:
Yes it is. It fixes an issue with user forms when built using lowest deps. I think it should be tackled in a ticket on its own.